.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,37 @@
-*To help us understand & test your amazing contribution, please include the below information with your PR. 🙏 Thanks!*
+<!-- Employees: Please use Clubhouse's "Open PR" button from the relevant story or include links to relevant Clubhouse stories in your branch name, commit messages, or pull request comments. Do not add links to your pull request description, they will be ignored. https://help.clubhouse.io/hc/en-us/articles/207540323-Using-The-Clubhouse-GitHub-Integration -->
 
-
-### Closes #ISSUE_NUMBER_GOES_HERE
+<!-- Employees: Delete this section. -->
+## Closes #ISSUE_NUMBER_GOES_HERE
 
 🎉 🎉 🎉
 
-### Description
+## Description
 
 ......
 
-
-### Changes
+## Changes
 
 * High level
 * changes that
 * you made
 
+### Reusable Code/Components
 
-### Steps to Test
+<!-- Preferably, include automated tests instead. -->
+## Steps to Test
 
 1. Steps
 2. to
 3. test
 
+<!-- Contributors: Delete this section. -->
+## [Zeplin Design](https://app.zeplin.io/project/5b0334f5e91e8c481645ad56)
+<!-- Upload screenshots here. -->
 
-### Screenshots
+## Quality Assurance
 
-...only if applicable...
+- [ ] The branch name is in lowercase-kebab-case with no prefix (unless it was created from Clubhouse)
```

Comment:
I'm for prefixing a PR branch with our initials. eg. `ev/sender-address` it has the benefit of making it's author responsible for it's branch and encourage us to reduce it's life span.

Reply:
I think this could get confusing when stories are reassigned between developers. You could always see the latest author of a particular branch with `git show`, and GitHub's branch page also includes authors if you want to prune branches.

Reply:
As you say there are multiple ways to find the author of a PR but the objective of prefixing is more to encourage short-lived branches. It's a small measure to encourage the right behaviour of small PR + quick merges + branch deletion.

If we're able to reach this objective, stories would no longer need to be reassigned because they will be completed before a reason occurs.

Reply:
I'm against adding any info other than Clubhouse linking to branch names honestly

Reply:
Ok let's try and see how it works